src/components/dropdownSearch/MultiSelectDropdownSearch/MultiSelectDropdownSearchConnected.tsx

```diff
@@ -0,0 +1,59 @@
+import { IReactVaporState, IReduxActionsPayload } from '../../../ReactVapor';
+import { IDropdownOption, IDropdownSearchOwnProps, IDropdownSearchProps, IDropdownSearchStateProps } from './../DropdownSearch';
+import { IReduxAction, ReduxUtils } from '../../../utils/ReduxUtils';
+import { connect } from 'react-redux';
+import * as _ from 'underscore';
+import {
+  addMultiSelectDropdownSearch,
+  applyFilterDropdownSearch,
+  multiSelectOptionDropdownSearch, openDropdownSearch, closeDropdownSearch,
+  removeAllSelectedOptionsMultiselectDropdownSearch,
+  removeDropdownSearch,
+  removeSelectedOptionDropdownSearch,
+  updateActiveOptionDropdownSearch, keyDownMultiselectDropdownSearch, addCustomSelectedOption,
+} from '../DropdownSearchActions';
+import { IDropdownSearchState } from './../DropdownSearchReducers';
+import { FixedQueue} from '../../../utils/FixedQueue';
+import { MultiSelectDropdownSearch } from './MultiSelectDropdownSearch';
+
+const mapStateToProps = (state: IReactVaporState, ownProps: IDropdownSearchProps): IDropdownSearchStateProps => {
+  const dropdownSearch: IDropdownSearchState = _.findWhere(state.dropdownSearch, { id: ownProps.id });
+
+  if (dropdownSearch) {
+    return {
+      isOpened: dropdownSearch.isOpened || false,
+      options: dropdownSearch.options || [],
+      displayedOptions: dropdownSearch.displayedOptions || dropdownSearch.options,
+      selectedOptions: dropdownSearch.selectedOptions,
+      filterText: dropdownSearch.filterText || '',
+      activeOption: dropdownSearch.activeOption,
+      setFocusOnDropdownButton: dropdownSearch.setFocusOnDropdownButton,
+    };
+  }
+
+  return {
+    isOpened: false,
+    options: ownProps.defaultOptions || [],
+    displayedOptions: ownProps.displayedOptions || ownProps.options,
+    selectedOptions: new FixedQueue<IDropdownOption>(),
+    filterText: '',
+  };
+};
+
+const mapDispatchToProps = (dispatch: (action: IReduxAction<IReduxActionsPayload>) => void,
```

Comment:
This needs the type of what it return (`MultiSelectDropdownSearchDispatchProps`)